CMakeLists.txt

```diff
@@ -142,7 +142,9 @@ if(NOT BlocksRuntime_FOUND)
                 ${PROJECT_SOURCE_DIR}/src/BlocksRuntime/runtime.c)
   set_target_properties(BlocksRuntime
                         PROPERTIES
-                          POSITION_INDEPENDENT_CODE TRUE)
+                          POSITION_INDEPENDENT_CODE TRUE
+                          C_VISIBILITY_PRESET HIDDEN
```

Comment:
Yeah, there is already a PR for that (and this was meant as a stop gap until we get that merged.  Since we discussed that we are going to actively pursue that first, this is no longer relevant.